packages/fe-fpm-writer/src/view/index.ts

```diff
@@ -51,9 +79,10 @@ function enhanceConfig(fs: Editor, data: CustomView, manifestPath: string, manif
  * @param {string} basePath - the base path
  * @param {CustomView} customView - the custom view configuration
  * @param {Editor} [fs] - the mem-fs editor instance
+ * @param {boolean} viewUpdate - indicates if th view fragment shall be created or overwritten
  * @returns {Promise<Editor>} the updated mem-fs editor instance
  */
-export function generateCustomView(basePath: string, customView: CustomView, fs?: Editor): Editor {
+export function generateCustomView(basePath: string, customView: CustomView, fs?: Editor, viewUpdate = true): Editor {
```

Comment:
Just small observation, I am only not sure if we should introduce new API parameter?
I think we usually use 2nd param to pass configuration, for example `overwrite` for `controller extension`:
https://github.com/SAP/open-ux-tools/blob/main/packages/fe-fpm-writer/src/controller-extension/types.ts#L39